packages/component-logger/server/logger.js

```diff
@@ -1,12 +1,14 @@
 const winston = require('winston')
+const emailFilter = require('./filters/emailFilter')
 
 // production logger sends JSON lines to file
-const logPath = process.env.XPUB_LOG_PATH || '/srv/elife-xpub/var/logs'
+const logPath = process.env.XPUB_LOG_PATH || '~/elife-xpub/var/logs'
```

Comment:
Are we sure we want to change this default?

Reply:
Thats a mistake it should be the old one